docs/serving/knative-kubernetes-services.md

```diff
@@ -82,10 +84,9 @@ and validates and mutates Kubernetes API calls.
 
 The certmanager reconciles cluster ingresses into cert manager objects.
 
-### Deployment: networking-istio
+### Deployment: networking-istio/contour-ingress-controller/etc.
```

Comment:
I wonder if this could be a generic title such as `Networking Layer` and all the supported networking layer deployment names could be included in a table